src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -415,4 +437,39 @@ std::pair<std::string, mediapipe_packet_type_enum> MediapipeGraphDefinition::get
     SPDLOG_LOGGER_DEBUG(modelmanager_logger, "setting input stream: {} packet type: {} from: {}", "", "UNKNOWN", streamFullName);
     return {"", mediapipe_packet_type_enum::UNKNOWN};
 }
+
+#if (PYTHON_DISABLE == 0)
+Status MediapipeGraphDefinition::initializeNodes() {
+    SPDLOG_INFO("MediapipeGraphDefinition initializing graph nodes");
+    for (int i = 0; i < config.node().size(); i++) {
+        if (config.node(i).calculator() == PYTHON_NODE_CALCULATOR_NAME) {
+            if (config.node(i).node_options().size()) {
```

Comment:
What if there is python node without options?

Reply:
I will add error then.